subprojects/configuration-cache/build.gradle.kts

```diff
@@ -5,20 +5,6 @@ plugins {
 
 description = "Configuration cache implementation"
 
-val configurationCacheReportPath by configurations.creating {
-    isVisible = false
-    isCanBeConsumed = false
-    attributes { attribute(DocsType.DOCS_TYPE_ATTRIBUTE, objects.named("configuration-cache-report")) }
-}
-
-dependencies {
-    configurationCacheReportPath(project(":configuration-cache-report"))
-}
-
-tasks.processResources {
-    from(configurationCacheReportPath) { into("org/gradle/configurationcache/problems") }
```

Comment:
Does this still work even though the configuration cache report is now on the classpath and not anymore in a resource folder? Should we make it the same as before and include the configuration cache report html in `configuration-cache.jar`?

Reply:
I don't understand this change. I'd have expected a change from project coordinates to `group:artifact` coordinates.

Reply:
Done.